### ISSUE TITLE
Adopt .editorconfig file and reformat

### ISSUE THREAD

**Issue by manuelmazzuola** [openstack4j/openstack4j#21]
Add .editorconfig to the repository root.  
This allows all contributors to have consistent code style.

**Comment by olivergondza** [comment]
:+1: The code style needs unification. Do you happen to know of a way to mass-modify all the codebase so it complies to that file? I have a strong preference of converting at once, so we do not have to run into ad-hoc PR reformats for years to come.

**Comment by manuelmazzuola** [comment]
> Do you happen to know of a way to mass-modify all the codebase so it complies to that file?

We can add the .edtiroconfig file to the root, all the code editors support it, intellij, vscode, eclipse etc., the editor will automatically read it and use the rules.

And then we can push a pr with all the code reformated with the editorconfig rules, on intellij you can do right click on the root folder and select `reformat code`.

**Comment by manuelmazzuola** [comment]
OR we can use https://checkstyle.org/ and integrate it into travis

**Comment by olivergondza** [comment]
We are currently discussing eventual merge back to ContainX/openstack4j and resurecting the project there. So I would like to postpone large scale changes to make merging back easier.

---

I have got this a try and I was able to generate fairly detailed .editorconfig with IDEA extension in it. Its enforcment can be added to maven so we get code checked with every build, there also is a way to get the code formatted from console. I like this.
